src/polyfills/stubs/intersection-observer-stub.js

```diff
@@ -260,20 +263,18 @@ export class IntersectionObserverStub {
   }
 
   /**
-   * @param {typeof IntersectionObserver} constr
+   * @param {!IntObsConstructorDef} Ctor
```

Comment:
nit: the shorthand i've seen used is `InOb` vs. `IntObs`

Reply:
Updated